app/helpers/issue_view_columns_issues_helper.rb

```diff
@@ -2,53 +2,242 @@
 
 module IssueViewColumnsIssuesHelper
   def render_descendants_tree(issue)
-    columns_list = get_fields_for_project issue
-    # no field defined, then use render from core redmine (or whatever by other plugins loaded before this)
-    return super if columns_list.count.zero?
+    # Retrieve the list of columns to display for the project
+    columns_list = get_fields_for_project(issue)
+    return super if columns_list.empty?
 
-    # continue here if there are fields defined
-    field_values = +''
-    s = table_start_for_relations columns_list
-    manage_relations = User.current.allowed_to? :manage_subtasks, issue.project
-    # set data
-    issue_list(issue.descendants.visible.preload(:status, :priority, :tracker, :assigned_to).sort_by(&:lft)) do |child, level|
-      next if child.closed? && !issue_columns_with_closed_issues?
+    # Retrieve minimum width settings for columns
+    min_width_setting = RedmineIssueViewColumns.setting(:columns_min_width)
+    min_widths = {}
+    if min_width_setting.present?
+      min_width_setting.split(",").each do |column_setting|
+        column_name, min_width = column_setting.split(":").map(&:strip)
+        min_widths[column_name] = min_width
+      end
+    end
 
-      tr_classes = +"hascontextmenu #{child.css_classes} #{cycle 'odd', 'even'}"
-      tr_classes << " idnt idnt-#{level}" if level.positive?
+    # Retrieve sorting settings and determine if sorting by directory/file model is enabled
+    sort_dir_file_model = RedmineIssueViewColumns.setting(:sort_dir_file_model)
+    collapsed_ids = issue.collapsed_ids.to_s.split.map(&:to_i)
```

Comment:
I'm curious, what is returned here if `collapsed_ids` is not completed (I think the default is `nil`, right?

Reply:
collapsed_ids will become an empty array and all the issues will be rendered normally